### PR TITLE
Add hh@cncf.io to kn-infra-gcp-org-admins

### PR DESCRIPTION
**Pull request by hh**
See: https://knative.slack.com/archives/CCSNR4FCH/p1650580357657709?thread_ts=1649809077.370369&cid=CCSNR4FCH

### CODE REVIEW

**Comment by hh** [comment]
https://prow.knative.dev/view/gs/knative-prow/pr-logs/pull/knative_community/1022/pull-knativeteam-groups-test/1517319296873664512#

```
--- FAIL: TestHardcodedGroupsForParanoia (0.00s)
[37](https://prow.knative.dev/view/gs/knative-prow/pr-logs/pull/knative_community/1022/pull-knativeteam-groups-test/1518772378019041280#1:build-log.txt%3A37)
    groups_test.go:301: group 'kn-infra-gcp-org-admins': expected members '[chizhg@google.com chizhg@knative.team cy@borg.dev cy@knative.team kmahapatra@vmware.com krsna@knative.team racker-maha-66b7d200@gcp.rackspace.com]', got '[chizhg@google.com chizhg@knative.team cy@borg.dev cy@knative.team hh@cncf.io hh@knative.team kmahapatra@vmware.com krsna@knative.team racker-maha-66b7d200@gcp.rackspace.com]'
```

Looking up TestHardcodedGroupsForParanoia! :)

**Comment by hh** [comment]
@csantanapr I think we are good now.

File: groups/groups_test.go [hunk 1]
```diff
@@ -282,6 +282,8 @@ func TestHardcodedGroupsForParanoia(t *testing.T) {
 			"cy@borg.dev",
 			"chizhg@knative.team",
 			"cy@knative.team",
+			"hh@cncf.io",
+			"hh@knative.team",
```
Comment (csantanapr) [inline groups/groups_test.go:286]: Now I see `Traking down TestHardCodedGRoupsForParanoia! (edited)`

**Pull request merged**